lib/api/src/js/exports.rs

```diff
@@ -53,6 +53,9 @@ pub enum ExportError {
     /// This error arises when an export is missing
     #[error("Missing export {0}")]
     Missing(String),
+    /// This error arises when an export is missing
+    #[error("Serialization failed {0}")]
+    SerializationFailed(String),
```

Comment:
Not sure if I understand when this can be triggered.
When an export can be missing?

Reply:
ah this is leftovers - this is not used anymore - we should remove it